File: frontend/src/app/shared/models/AdminModel/EvaluationFormModel/AdminResponseEvaluationFormDto.ts
export interface AdminResponseEvaluationFormDto {
  id:number,
  division: string,
  agreementLevel: number|null,
  perceivedRisk: string|null,
  certaintyLevel: number|null,
  comment: string | null,
  username: string,
  firstName: string,
  lastName: string,
  isPublic: boolean
}

File: backend/src/main/java/com/webgis/admin/dto/evaluationform/AdminResponseEvaluationFormDto.java
package com.webgis.admin.dto.evaluationform;

import com.webgis.evaluationform.EvaluationForm;

/**
 * Data Transfer Object containing all the evaluation form information for admin
 */
public record AdminResponseEvaluationFormDto(
        long id,
        String division,
        Integer agreementLevel,
        String perceivedRisk,
        Integer certaintyLevel,
        String comment,
        String username,
        String firstName,
        String lastName,
        boolean isPublic
) {
    public AdminResponseEvaluationFormDto(EvaluationForm evaluationForm) {
        this(
                evaluationForm.getId(),
                evaluationForm.getDivision(),
                evaluationForm.getAgreementLevel(),
                evaluationForm.getPerceivedRisk(),
                evaluationForm.getCertaintyLevel(),
                evaluationForm.getComment(),
                evaluationForm.getUser().getUsername(),
                evaluationForm.getUser().getFirstName(),
                evaluationForm.getUser().getLastName(),
                evaluationForm.isPublic()
        );
    }
}